02_root_bootargs_grub.cfg

```diff
@@ -0,0 +1 @@
+set rootargs="rootwait ${mender_root}"
```

Comment:
With this change, `mender_root` needs to be defined much earlier. Is there a reason we can't leave it where it is, and keep `rootwait` separate?

In any case, I think you missed `root=` here.

Reply:
> With this change, mender_root needs to be defined much earlier. Is there a reason we can't leave it where it is, and keep rootwait separate?

Just an optimization, as we keep the "root=" separate from "bootargs" currently. But will update the PR and keep the "root=" as is and add 'rootwait' separately.